### PR TITLE
Create Minesweeper page

### PR DESCRIPTION
**Pull request by danielbk08**

### CODE REVIEW

**Comment by sandyandi** [comment]
Please pull latest `master` branch and use blade templates.

File: app/Http/Controllers/MinesweeperController.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Created by PhpStorm.
+ * User: daniel
+ * Date: 3/7/17
+ * Time: 14:56
+ */
+
+namespace App\Http\Controllers;
+
+
```
Comment (sandyandi) [inline app/Http/Controllers/MinesweeperController.php]: Remove duplicate blank line.

File: app/Http/Controllers/MinesweeperController.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+class MinesweeperController extends Controller
+{
+    public function index(){
```
Comment (sandyandi) [inline app/Http/Controllers/MinesweeperController.php]: Please follow PSR-2 standards.

File: resources/views/pages/minesweeper.blade.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@extends('master')
+
+@section('content')
+
+<h1 class="text-center">Minesweeper</h1>
+<div class="col-xs-6 col-md-offset-3 text-center mt-400">
+    <table id="table_div" class= "table table-responsive table-bordered">
+        <thead></thead>
+        <tbody>
+            <?php for($i = 1; $i < 9 ; $i++){ ?>
```
Comment (sandyandi) [inline resources/views/pages/minesweeper.blade.php]: Please use [Blade directives](https://laravel.com/docs/5.4/blade).

File: routes/web.php [hunk 4]
```diff
@@ -17,8 +17,10 @@
 
 Route::get('/contact', 'ContactController@index');
 
-Route::get('astroport', function () {
-    return view('pages.astroport');
+Route::get('/minesweeper', 'MinesweeperController@index');
+
+Route::get('/contact', function () {
```
Comment (sandyandi) [inline routes/web.php]: Remove duplicate route.

File: routes/web.php [hunk 5]
```diff
@@ -17,8 +17,10 @@
...
+    return view('pages.contact');
 });
 
 Route::get('/primeFactors', 'PrimeFactorsController@index');
```
Comment (sandyandi) [inline routes/web.php]: Please follow PSR standards (empty newline at the end of the file).

File: tests/Feature/MinesweeperTest.php [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+/**
+ * Created by PhpStorm.
+ * User: daniel
+ * Date: 3/7/17
+ * Time: 15:02
+ */
+
+namespace Tests\Feature;
+
+
```
Comment (sandyandi) [inline tests/Feature/MinesweeperTest.php]: Remove duplicate empty line.

File: tests/Feature/MinesweeperTest.php [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+use Tests\TestCase;
+
+class MinesweeperTest extends TestCase
```
Comment (sandyandi) [inline tests/Feature/MinesweeperTest.php]: Please follow PSR standards.

File: resources/views/pages/minesweeper.blade.php [hunk 8]
```diff
@@ -0,0 +1,21 @@
+@extends('master')
+
+@section('content')
+
+<h1 class="text-center">Minesweeper</h1>
+<div class="col-xs-6 col-md-offset-3 text-center mt-400">
+    <table id="table_div" class= "table table-responsive table-bordered">
```
Comment (ToshieUya) [inline resources/views/pages/minesweeper.blade.php]: @danielbk08 why use table bruh? use flex! haha :P

**Pull request merged**